### PR TITLE
[V2]: Properties are now camelCased instead of hyphenated for better compatibility.

### PR DESCRIPTION
**Pull request by bhough**
Outputted properties are now camelCased instead of hyphenated. Modules that accept property names can accept both (would possibly want to deprecate this later). Documentation has been updated to reflect these changes.

Addresses: #167
~***Blocked by : #184***~

### CODE REVIEW

**Comment by bhough** [comment]
Related `styled-components` issue https://github.com/styled-components/styled-components/issues/787

**Comment by bhough** [comment]
@mxstbr @nikgraf this is all cleaned up and ready to review now that #184 has been merged into `version-2`

File: src/mixins/normalize.js [hunk 1]
```diff
@@ -281,13 +280,11 @@ function mergeRules(baseRules: Object, additionalRules: Object) {
  * // CSS as JS Output
  *
  * html {
- *   font-family: sans-serif,
- *   line-height: 1.15,
- *   -ms-text-size-adjust: 100%,
- *   -webkit-text-size-adjust: 100%,
+ *   fontFamily: sans-serif,
```
Comment (nikgraf) [inline src/mixins/normalize.js]: I'm confused about this one. If it's JS output it `sans-serif` should be wrapped in String?!?

File: src/shorthands/transitions.js [hunk 2]
```diff
@@ -10,7 +10,7 @@
  *
  * // styled-components usage
  * const div = styled.div`
- *   ${...transitions('opacity 1.0s ease-in 0s', 'width 2.0s ease-in 2s')}
```
Comment (nikgraf) [inline src/shorthands/transitions.js]: was this a bug in our docs?

File: src/mixins/normalize.js (hunk 1 above)
Comment (bhough) [inline src/mixins/normalize.js]: yup looks like it was wrong from the start and just needs fixed.

File: src/shorthands/transitions.js (hunk 2 above)
Comment (bhough) [inline src/shorthands/transitions.js]: yes it was.

**Comment by emattias** [comment]
Can you publish a v2 branch to npm? I need it just to avoid glamor warnings all over the place! :)

**Comment by bhough** [comment]
@emattias ✨ polished 1.3 moves to camel-casing and is out now: https://github.com/styled-components/polished/releases/tag/v1.3.0

**Pull request merged**